src/chain/mod.rs

```diff
@@ -915,6 +915,21 @@ impl ChainSource {
 								"Failed to retrieve fee rate estimates: {}. Falling back to defaults.",
 								e,
 							);
+
+							let unix_time_secs_opt = SystemTime::now()
+								.duration_since(UNIX_EPOCH)
+								.ok()
+								.map(|d| d.as_secs());
+							{
+								let mut locked_node_metrics = node_metrics.write().unwrap();
+								locked_node_metrics.latest_fee_rate_cache_update_timestamp =
+									unix_time_secs_opt;
+								write_node_metrics(
+									&*locked_node_metrics,
+									Arc::clone(&kv_store),
+									Arc::clone(&logger),
+								)?;
+							}
```

Comment:
`LDKNode.swift` changes look out of place in d6b689494161164e296e50f70476b3eea44c25b9.